sdk/core/System.ClientModel/api/System.ClientModel.net6.0.cs

```diff
@@ -67,6 +83,19 @@ public partial class ClientPipelineOptions
         public System.ClientModel.Primitives.PipelineTransport? Transport { get { throw null; } set { } }
         public void AddPolicy(System.ClientModel.Primitives.PipelinePolicy policy, System.ClientModel.Primitives.PipelinePosition position) { }
     }
+    public partial class ClientRetryPolicy : System.ClientModel.Primitives.PipelinePolicy
+    {
+        public ClientRetryPolicy() { }
+        public ClientRetryPolicy(int maxRetries, System.ClientModel.Primitives.PipelineMessageDelay delay) { }
```

Comment:
TODO: is it easy to remove MessagDelay in favor of configurable properties in this constructor?

Reply:
Investigating here: https://github.com/Azure/azure-sdk-for-net/pull/41358

Reply:
Addressed